App/src/Controller/RegisterController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use App\Form\UserSignInType;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
+use Symfony\Component\Routing\Annotation\Route;
+
+class RegisterController extends AbstractController
+{
+    #[Route('/signin', name: 'signin', methods: ['GET', 'POST'])]
```

Comment:
"Sign in" ou "Log in" c'est plutôt "s'authentifier" ou "se connecter".
"Sign up" c'est "s'enregistrer" ou "créer un compte"

Reply:
https://ell.stackexchange.com/questions/24384/what-is-the-difference-among-sign-up-sign-in-and-log-in

Reply:
Done, j'ai modifié toutes les occurrences dans mon code pour être bien au clair partout du coup